src/api/community-api/search.ts

```diff
@@ -0,0 +1,32 @@
+import axios from 'axios';
+import type { CommunityPost, CommunityTopic } from '../../types';
+
+import env from '../../env';
+import { CommunityTerm } from '../../types/enums';
+
+const { FDK_COMMUNITY_BASE_URI } = env;
+
+export const searchCommunity = (queryTerm: string) =>
+  axios
+    .get(`${FDK_COMMUNITY_BASE_URI}/api/search?term=${queryTerm}`)
+    .then(response => response.data);
```

Comment:
Her og tilsvarende kommentar andre steder i denne filen

```suggestion
    .then(({ data }) => data);
```